gcloud-java-contrib/gcloud-java-nio/src/test/java/com/google/cloud/storage/contrib/nio/it/ITGcsNio.java

```diff
@@ -1,3 +1,18 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
Yes. I copied this from one of the other files, so if we care about this we'll probably want a PR with a global search and replace sometime.